components/base/Badge.tsx

```diff
@@ -24,6 +19,25 @@ const StyledBadge = styled.div`
   border: 1px solid ${GREY_4};
   border-radius: 10px;
   padding: 2px 5.15px;
+  &.orange {
+    background-color: ${ORANGE_PRIMARY};
+    border-color: ${ORANGE_PRIMARY};
+    border-radius: 15px;
+    color: ${WHITE_PRIMARY};
+    font-size: 13px;
+    letter-spacing: 0;
+    height: 22px;
+    padding: 2px 8px;
+  }
 `;
 
+type Props = { text: string; variant?: "default" | "orange" };
+
+function Badge({ text, variant }: Props) {
+  return (
+    <StyledBadge className={classnames({ orange: variant })}>
```

Comment:
I think we just want this just be `className={ variant }`

As it is now, it'll apply the `.orange` class whenever variant is truthy, which is the case even if variant is set to "default"

Reply:
Ah, good catch! 👍